apps/prairielearn/src/lib/manualGrading.ts

```diff
@@ -318,13 +318,13 @@ export async function updateAssessmentQuestionRubric(
           number,
           rubric_id: new_rubric_id,
         })),
-        async (item) => {
+        async ({ order: _order, id, ...item }) => {
           // Attempt to update the rubric item based on the ID. If the ID is not set or does not
           // exist, insert a new rubric item.
-          if (item.id == null) {
-            await sqldb.queryAsync(sql.insert_rubric_item, _.omit(item, ['order', 'id']));
+          if (id == null) {
+            await sqldb.queryAsync(sql.insert_rubric_item, item);
```

Comment:
Nit, I actually don't mind `_.omit`, the alternative (destructuring with unused properties) looks a little weird IMO. This is subjective though.

Reply:
Do you think I should remove the _.omit error in the eslint config (in essence revert a907d2d685f7f8b183d4476f9ecfa4b74b950d49)?

Reply:
I lean towards doing that, yeah.

Reply:
Done in af5f177f93a2edf056326ff0bf1ec1a0e1f15f62.